### PR TITLE
feat(frontend): add label input on settings parameters page

### PR DESCRIPTION
**Pull request by charline-rageade**
#4555 

## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [x] Manual, quality tests have been done

### CODE REVIEW

**Comment by charline-rageade** [comment]
Automatic tests do not seem necessary for this minor modification.

File: frontend/src/component/Account/PersonalData.jsx [hunk 1]
```diff
@@ -63,25 +64,31 @@ export class PersonalData extends React.Component {
...
-            <input
-              className='personaldata__form__txtinput primaryColorBorderLighten form-control'
-              type='text'
-              data-cy='personaldata__form__txtinput__fullname'
-              placeholder={props.t('New full name')}
-              value={state.newPublicName}
-              onChange={this.handleChangePublicName}
-            />
+            <label>
+              {props.t('New full name')}
```
Comment (raphj) [inline frontend/src/component/Account/PersonalData.jsx:68]:
```suggestion
              {props.t('New full name:')&nbsp;
```
you will need to update the translation strings too

**Review by raphj — CHANGES_REQUESTED** [review]
:+1:

File: frontend/i18next.scanner/fr/translation.json [hunk 2]
```diff
@@ -454,5 +451,8 @@
   "Download {{filename}}": "Télécharger {{filename}}",
   "Show in publications": "Afficher dans les publications",
   "This space does not have any publication yet, create the first publication using the area at the bottom of the page.": "Cet espace n'a pas encore de publication, créez la première publication en utilisant la zone en bas de la page.",
-  "Open as content": "Ouvrir en tant que contenu"
+  "Open as content": "Ouvrir en tant que contenu",
+  "New full name:": "Nouveau nom complet:",
+  "New username:": "Nouvel identifiant:",
+  "New email:": "Nouvel email:"
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/fr/translation.json:457]:
```suggestion
  "New full name:": "Nouveau nom complet :",
  "New username:": "Nouvel identifiant :",
  "New email:": "Nouvel email :"
```

File: frontend/i18next.scanner/pt/translation.json [hunk 3]
```diff
@@ -454,5 +451,8 @@
   "Download {{filename}}": "Download {{filename}}",
   "Show in publications": "Mostrar em publicações",
   "This space does not have any publication yet, create the first publication using the area at the bottom of the page.": "Este espaço ainda não tem nenhuma publicação, crie a primeira publicação utilizando a zona na parte de baixo da página.",
-  "Open as content": "Abrir como conteúdo"
+  "Open as content": "Abrir como conteúdo",
+  "New full name:": "Novo nome completo :",
+  "New username:": "Novo identificador :",
+  "New email:": "Novo email :"
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:457]:
```suggestion
  "New full name:": "Novo nome completo:",
  "New username:": "Novo identificador:",
  "New email:": "Novo e-mail:"
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend/src/component/Account/PersonalData.jsx [hunk 4]
```diff
@@ -144,6 +157,9 @@ PersonalData.propTypes = {
 }
 
 PersonalData.defaultProps = {
+  userEmail: '',
+  userUsername: '',
+  userPublicName: '',
```
Comment (raphj) [inline frontend/src/component/Account/PersonalData.jsx:162]:
```suggestion
```

File: frontend/src/component/Account/PersonalData.jsx [hunk 5]
```diff
@@ -136,6 +146,9 @@ export class PersonalData extends React.Component {
 }
 
 PersonalData.propTypes = {
+  userEmail: PropTypes.string,
+  userUsername: PropTypes.string,
+  userPublicName: PropTypes.string,
```
Comment (raphj) [inline frontend/src/component/Account/PersonalData.jsx:151]:
```suggestion
  userEmail: PropTypes.string.isRequired,
  userUsername: PropTypes.string.isRequired,
  userPublicName: PropTypes.string.isRequired,
```

**Review by raphj — APPROVED** [review]

File: frontend/src/component/Account/PersonalData.jsx (hunk 5 above)
Comment (charline-rageade) [reply frontend/src/component/Account/PersonalData.jsx:151]: we don't need the .isRequired in this case

**Review by gbonaspetti — APPROVED** [review]

File: frontend/i18next.scanner/fr/translation.json [hunk 6]
```diff
@@ -454,5 +451,8 @@
   "Download {{filename}}": "Télécharger {{filename}}",
   "Show in publications": "Afficher dans les publications",
   "This space does not have any publication yet, create the first publication using the area at the bottom of the page.": "Cet espace n'a pas encore de publication, créez la première publication en utilisant la zone en bas de la page.",
-  "Open as content": "Ouvrir en tant que contenu"
+  "Open as content": "Ouvrir en tant que contenu",
+  "New full name:": "Nouveau nom complet :",
+  "New username:": "Nouvel identifiant :",
+  "New email:": "Nouvel email :"
```
Comment (PhilippeAccorsi) [inline frontend/i18next.scanner/fr/translation.json:457]: Ce sont des espaces insécable avant le `:` ? (c'est mieux pour le responsive il me semble)

**Comment by gbonaspetti** [comment]
After tests:
- The password doesn't have a label
Screenshot from 2021-04-27 14-21-43

- It would be nice if the password page in the menu also had labels to be consistent with the "my account" page
Screenshot from 2021-04-27 14-21-33

File: frontend/i18next.scanner/pt/translation.json [hunk 7]
```diff
@@ -454,5 +446,13 @@
   "Download {{filename}}": "Download {{filename}}",
   "Show in publications": "Mostrar em publicações",
   "This space does not have any publication yet, create the first publication using the area at the bottom of the page.": "Este espaço ainda não tem nenhuma publicação, crie a primeira publicação utilizando a zona na parte de baixo da página.",
-  "Open as content": "Abrir como conteúdo"
+  "Open as content": "Abrir como conteúdo",
+  "New full name:": "Novo nome completo:",
+  "New username:": "Novo identificador:",
+  "New email:": "Novo e-mail:",
+  "New password:": "nova palavra-passe:",
+  "Repeat new password:": "Repetir nova palavra-passe",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:454]:
```suggestion
  "Repeat new password:": "Repetir nova palavra-passe:",
```

File: frontend/i18next.scanner/pt/translation.json [hunk 8]
```diff
@@ -454,5 +446,13 @@
   "Download {{filename}}": "Download {{filename}}",
   "Show in publications": "Mostrar em publicações",
   "This space does not have any publication yet, create the first publication using the area at the bottom of the page.": "Este espaço ainda não tem nenhuma publicação, crie a primeira publicação utilizando a zona na parte de baixo da página.",
-  "Open as content": "Abrir como conteúdo"
+  "Open as content": "Abrir como conteúdo",
+  "New full name:": "Novo nome completo:",
+  "New username:": "Novo identificador:",
+  "New email:": "Novo e-mail:",
+  "New password:": "nova palavra-passe:",
```
Comment (gbonaspetti) [inline frontend/i18next.scanner/pt/translation.json:453]:
```suggestion
  "New password:": "Nova palavra-passe:",
```

**Review by gbonaspetti — CHANGES_REQUESTED** [review]

File: frontend/src/component/Account/Password.jsx [hunk 9]
```diff
@@ -95,58 +98,70 @@ export class Password extends React.Component {
...
+          <div className='d-flex align-items-center flex-wrap mb-4'>
+            <label>
+              {props.t('New password:')}
               <input
                 className='personaldata__form__txtinput primaryColorBorderLighten form-control'
                 type='password'
-                placeholder={props.t('Old password')}
-                value={state.oldPassword}
-                onChange={this.handleChangeOldPassword}
+                placeholder=''
```
Comment (raphj) [inline frontend/src/component/Account/Password.jsx:121]:
```suggestion
```

File: frontend/src/component/Account/Password.jsx [hunk 10]
```diff
@@ -95,58 +98,70 @@ export class Password extends React.Component {
...
+              {props.t('Repeat new password:')}
               <input
-                className='personaldata__form__txtinput checkPassword primaryColorBorderLighten form-control mt-3'
+                className='personaldata__form__txtinput withAdminMsg primaryColorBorderLighten form-control'
                 type='password'
-                placeholder={props.t("Administrator's password")}
-                value={state.checkAdminPassword}
-                onChange={this.handleChangeCheckAdminPassword}
-                disabled={state.newPassword === '' && state.newPassword2 === ''}
+                placeholder=''
```
Comment (raphj) [inline frontend/src/component/Account/Password.jsx:135]:
```suggestion
```

File: frontend/src/component/Account/Password.jsx [hunk 11]
```diff
@@ -95,58 +98,70 @@ export class Password extends React.Component {
...
+            </label>
+          </div>
+          <div className=' d-flex align-items-center flex-wrap mb-4'>
+            {props.displayAdminInfo && state.newPassword !== '' && (
+              <label>
+                {props.t("Administrator's password:")}
+                <input
+                  className='personaldata__form__txtinput checkPassword primaryColorBorderLighten form-control mt-3'
+                  type='password'
+                  placeholder=''
```
Comment (raphj) [inline frontend/src/component/Account/Password.jsx:149]:
```suggestion
```

File: frontend/src/component/Account/PersonalData.jsx [hunk 12]
```diff
@@ -120,25 +123,28 @@ export class PersonalData extends React.Component {
...
-                value={state.checkPassword}
-                onChange={this.handleChangeCheckPassword}
-                disabled={state.newEmail === '' && state.newUsername === ''}
-              />
+              <label>
+                {props.displayAdminInfo ? props.t("Administrator's password:") : props.t('Type your password:')}
+                <input
+                  className='personaldata__form__txtinput checkPassword primaryColorBorderLighten form-control mt-3 mt-sm-0'
+                  type='password'
+                  placeholder=''
```
Comment (raphj) [inline frontend/src/component/Account/PersonalData.jsx:131]:
```suggestion
```

**Review by raphj — APPROVED** [review]

File: frontend/src/component/Account/Password.jsx (hunk 9 above)
Comment (charline-rageade) [reply frontend/src/component/Account/Password.jsx:121]: @raphj thanks i didn't know if i should delete it or not :)

**Review by gbonaspetti — APPROVED** [review]

File: functionnal_tests/cypress/integration/account/content_account_spec.js [hunk 13]
```diff
@@ -2,6 +2,8 @@ import { PAGES } from '../../support/urls_commands'
 import { SELECTORS as s } from '../../support/generic_selector_commands'
 import baseUser from '../../fixtures/baseUser.json'
 
+
+
```
Comment (raphj) [inline functionnal_tests/cypress/integration/account/content_account_spec.js:6]: This is probably too many lines (would not pass `standard`'s linting, but we do not currently lint Cypress files).

The following suggestion would fix it but feel free to ignore, we will probably clean up all the linting issues in cypress files when we are ready to do it.

```suggestion


```

**Review by raphj — APPROVED** [review]

**Comment by MariePrandstetter** [comment]
tested on branch

**Pull request merged**